server/repository/file_pairs.go

```diff
@@ -0,0 +1,42 @@
+package repository
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/src-d/code-annotation/server/model"
+)
+
+// FilePairs repository
+type FilePairs struct {
+	DB *sql.DB
```

Comment:
maybe it should be private?

Reply:
DB is initialized in router.go. Or should it be a private field with a setter method?

Reply:
usually, people do it with a constructor